dm/ctl/master/operate_task.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"errors"
+	"os"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+)
+
+const (
+	batchSizeFlag    = "batch-size"
+	defaultBatchSize = 5
+)
+
+type batchTaskResult struct {
+	Result       bool                 `json:"result"`
+	Msg          string               `json:"msg"`
+	BatchResults []*operateTaskResult `json:"batchResults"`
```

Comment:
```suggestion
	Tasks []*operateTaskResult `json:"tasks"`
```
It's similar to `sources`. `batchResults` is a little confusing to me.